frontend/micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSAddV2.js

```diff
@@ -43,7 +44,7 @@ const MDMSAdd = ({ defaultFormData, updatesToUISchema, screenType = "add", onVie
     params: {},
     body: {
       SchemaDefCriteria: {
-        tenantId: tenantId || stateId,
```

Comment:
tenantid has more priority , so when a useer at city a , he adds data to city a
and in search screen he sees data in city a

Reply:
updated